config/default.yml

```diff
@@ -1255,6 +1255,12 @@ Lint/ErbNewArguments:
   Enabled: true
   VersionAdded: '0.56'
 
+Lint/FlipFlop:
+  Description: 'Checks for flip-flops'
+  StyleGuide: '#no-flip-flops'
+  Enabled: true
+  VersionAdded: '0.16'
```

Comment:
@bbatsov Is it necessary to add `VersionChanged` metadata when moving a department? I have not added `VersionChanged: '0.63'` because there is no behavior change of this cop.

Reply:
I guess that's fine.